sklearn/linear_model/base.py

```diff
@@ -368,7 +368,7 @@ class LinearRegression(LinearModel, RegressorMixin):
     fit_intercept : boolean, optional, default True
         whether to calculate the intercept for this model. If set
         to False, no intercept will be used in calculations
-        (e.g. data is expected to be already centered).
+        (i.e. data is expected to be already centered).
```

Comment:
waiting for others input but you may want to not fit the intercept for various reasons, having already centered data is just one of them so maybe 'e.g.' still makes sense?

Reply:
Well, when you do not want to fit the intercept that should be because you _assume_ the data to be centered for several reasons. Maybe you show this on quantitative, maybe theoretic basis.
In this context I understand "centering" not as an action (some preprocessing which has been done to the data) but instead a property the data is assumed to have (for whatever reason, _maybe_ because of preprocessing).
But there is the "already" which totally proves your point, I did not read that one carefully before.
Maybe we could leave out the "already" and stress the data property?
Somehow the way it is formulated was misleading me, maybe I am the only one and then I should stay quiet - natural languages are tricky ;-)

Reply:
Can you come up with other wording? Note that this same phrasing is used 26 times in the repo.

Reply:
What about:

    (i.e. data is expected to be centered).